docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+version: '3'
+services:
+
+  mysql:
+    container_name: mysql
+    image: amd64/mysql:5.7.37
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+    volumes:
+      - ./mysql/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+      - mysql_data:/var/lib/mysql
+
+  elasticsearch:
+    container_name: elasticsearch
+    hostname: elasticsearch
+    build: ./elasticsearch/.
+    environment:
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "discovery.type=single-node"
+      - xpack.security.enabled=false  # override what is in elasticsearch.yml
+      - bootstrap.memory_lock=true
+      #- ELASTIC_PASSWORD=ElasticRocks  # default username is elastic
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - ./elasticsearch/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+      - es_data:/usr/share/elasticsearch/data
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+
+  kibana:
+    container_name: kibana
+    image: docker.elastic.co/kibana/kibana:7.16.2
+    ports:
+      - 5601:5601
+
+  chorus-ui:
+    container_name: chorus_es_ui
+    build: ./reactivesearch/.
+    volumes:
+      - './reactivesearch:/usr/src/app'
+      - '/usr/src/app/node_modules'
+    ports:
+      - 4001:3000
+
+  redis:
+    container_name: quepid_redis
+    image: redis:6.2.6-alpine
+    ports:
+      - 6379:6379
+
+  quepid:
+    container_name: quepid
+    image: o19s/quepid:6.10.0
+    ports:
+      - 3000:3000
+    command: "foreman s -f Procfile"
+    environment:
+      - PORT=3000
+      - RACK_ENV=production
+      - RAILS_ENV=production
+      - DATABASE_URL=mysql2://root:password@mysql:3306/quepid
+      - REDIS_URL=redis://redis:6379/1
+      - FORCE_SSL=false
+      - MAX_THREADS=2
+      - WEB_CONCURRENCY=2
+      - RAILS_LOG_TO_STDOUT=true
+      - RAILS_SERVE_STATIC_FILES=true
+      - SECRET_KEY_BASE=chorus_key
+      - TC_URL=
+      - PRIVACY_URL=
+      - COOKIES_URL=
+      - EMAIL_MARKETING_MODE=false
+      - EMAIL_PROVIDER=
+      - QUEPID_DEFAULT_SCORER=DCG@10
+      - KEYCLOAK_REALM=chorus
+      - KEYCLOAK_SITE=http://keycloak:9080
+      - SIGNUP_ENABLED=true
+    links:
+      - mysql
+      - redis
+    depends_on:
+      - mysql
+      - redis
+
+  prometheus:
+    image: prom/prometheus:v2.32.1
+    container_name: prometheus
+    restart: always
+    hostname: prometheus
+    ports:
+      - 9090:9090
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+
+  grafana:
+    image: grafana/grafana:7.5.13
+    container_name: grafana
```

Comment:
Not needed

Reply:
Kept `container_name` to have a clear container name and not a generated one.